src/main/entities.ts

```diff
@@ -0,0 +1,14 @@
+import LoggerService from "./services/logger.service";
+import AppService from "./services/app.service";
+
+
+/**
+ * List of classes what you want to be autoInjectable
+ *
+ * Like angular DI
+ *
+ */
+export default [
+    LoggerService,
```

Comment:
For consistency with the rest of the codebase, please keep indentation to two spaces (here and in other places).